pages/task/read/views/components/activity-log.jsx

```diff
@@ -27,15 +27,19 @@ function Action({ task, action, changedBy }) {
   );
 }
 
-function InspectorRecommendation({ item, task }) {
+function InspectorRecommendation({ item }) {
   if (!['inspector-recommended', 'inspector-rejected'].includes(item.status)) {
     return null;
   }
 
-  const isExtended = get(task, 'data.deadline.isExtended', false);
-  const deadline = get(task, 'data.deadline');
-  const deadlineDate = get(deadline, isExtended ? 'extended' : 'standard');
-  const daysSinceDeadline = daysSinceDate(deadlineDate, item.createdAt);
+  const deadlineAtTimeOfRecommendation = get(item, 'event.data.deadline');
+  let daysSinceDeadline;
+
+  if (deadlineAtTimeOfRecommendation) {
+    const isExtended = get(deadlineAtTimeOfRecommendation, 'isExtended', false);
+    const deadlineDate = get(deadlineAtTimeOfRecommendation, isExtended ? 'extended' : 'standard');
+    daysSinceDeadline = daysSinceDate(deadlineDate, item.createdAt);
```

Comment:
The "days since deadline" decoration does not get applied to the task data in activity items, so we still have to calc it here for the message to display.